docs/guides/stay-up-to-date/releases/2026-07/index.md

```
@@ -0,0 +1,552 @@
+<p style={{marginTop: "-20px", marginBottom: "10px"}}>
+  <small>Published: 7/14/2026 | <a href="https://github.com/gruntwork-io/pipelines-workflows/releases/tag/v4.21.0">Release notes</a></small>
+</p>
+
+<div style={{"overflow":"hidden","textOverflow":"ellipsis","display":"-webkit-box","WebkitLineClamp":10,"lineClamp":10,"WebkitBoxOrient":"vertical"}}>
+
+  
+
+GitHub is [rolling out a new format](https://github.blog/changelog/2026-04-23-immutable-subject-claims-for-github-actions-oidc-tokens/) for the OIDC `sub` used by Pipelines to authenticate to the cloud. Repositories created after July 15, 2026 will use this new format. In order to use the correct format when Account Factory creates new delegated repositories, upgrade to this Pipelines release along with [terraform-aws-architecture-catalog v6.1.2](https://github.com/gruntwork-io/terraform-aws-architecture-catalog/releases/tag/v6.1.2) and [terraform-aws-security v1.5.1](https://github.com/gruntwork-io/terraform-aws-security/releases#release-v1.5.1) or newer.
+
+
+Hooks can now specify a `filter` attribute to scope hooks to specific units. Hooks can be filtered by environment, label, and path. Hooks with a `filter` will only run when their targeted units are planned/applied. Read more [here](https://docs.gruntwork.io/2.0/docs/pipelines/guides/hooks/configuring/#scoping-a-hook-to-specific-units).
```

markdownlint-cli2 (0.23.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)